.gitlab-ci.yml

```diff
@@ -247,15 +247,15 @@ regression-composer-works-behind-satellite-fallback:
   extends: .regression
   rules:
     # BLACKLIST: Skipped on subscribed RHEL machines
```

Comment:
Is this still valid?
It skips all except nightly?

Reply:
I believe so, it's been like this ever since, here is a commit from 2 years ago that setup these rules and removed them from the scripts. https://github.com/osbuild/osbuild-composer/commit/1873c561cf799c0cb3552361b8f7e7e4745830d2

Reply:
I was just wondering if we should update the comment to match the implementation?
The comment should describe the condition, right? Is this the functionality that "nightly" is subscribed and the other aren't?